Makefile

```
@@ -47,7 +47,12 @@ destroy-box:
 	vagrant destroy -f
 
 remove-tmp:
-	rm -rf ./tmp
+    rm -rf ./tmp
+    rm -rf ./.vagrant
+    rm -rf ./.minio.sys
+    rm -rf ./example/.terraform
+    rm -rf ./example/terraform.tfstate
+    rm -rf ./example/terraform.tfstate.backup
```

Tests failed because of formatting. The solution provided in this page is reccomended: https://stackoverflow.com/questions/16931770/makefile4-missing-separator-stop
This command detects any missing tabs so good to use : ```cat -e -t -v makefile_name```
```suggestion
remove-tmp:
	rm -rf ./tmp
	rm -rf ./.vagrant
	rm -rf ./.minio.sys
	rm -rf ./example/.terraform
	rm -rf ./example/terraform.tfstate
	rm -rf ./example/terraform.tfstate.backup
```

---

In order to fix this tabbing issue in intellij, select the above block of code and then :
Ctrl + Shift + A > write "tabs" > double click on "To Tabs"